owlbot.py

```diff
@@ -125,6 +103,7 @@ def install_alembic_for_python_38(session, constraints_path):
 
 s.replace( ["noxfile.py"], old_sessions, new_sessions)
 
+# Maybe we can get rid of this when we don't need pytest-rerunfailures
```

Comment:
Maybe add a link to query retry issue/pr?

Reply:
OK

Reply:
Done